hugolib/site.go

```diff
@@ -105,7 +106,8 @@ type SiteInfo struct {
 	Authors               AuthorList
 	Social                SiteSocial
 	Sections              Taxonomy
-	Pages                 *Pages
+	Pages                 *Pages // Includes only pages in this language
+	TranslatedPages       *Pages // Includes other translated pages, excluding those in this language.
```

Comment:
Not sure how useful TranslatedPages are. I would prefer AllPages; then I could filter with a where clause if needed.